network/manager.go

```diff
@@ -867,3 +882,56 @@ func generateCNSIPInfoMap(eps []*endpoint) map[string]*restserver.IPInfo {
 
 	return ifNametoIPInfoMap
 }
+
+// GetEndpointInfos gets all endpoint infos associated with a container id
```

Comment:
this comment will be confused with the other method `GetEndpointInfosFromContainerID`. We pass in more than the container id here (and it isn't obvious that the container id is even passed in). would prefer  we pass in each of the args that we use separately as parameters since we only use ContainerID, Netns, and IfName. comment should also explain what each of the parameters is used for and how it differs from `GetEndpointInfosFromContainerID`.